### ISSUE TITLE
NullPointerException when calculating times

### ISSUE THREAD

**Issue by medwards** [conveyal/gtfs-editor#178]
Created a series of stops, and received this error when:
* There was no travel/dwell time specified on each stop
* When there was a travel/dwell time specified for every stop

I was unable to test what happens with an alignment due to #176

**Comment by medwards** [comment]
Once an alignment is created (blocked by #176 and fixed in #187), an alignment is successfully created (but the dwell time isn't populated properly). Should we change this to be about dwell times or close it and start a new issue?
